spec.toml

```diff
@@ -0,0 +1,3 @@
+server_url = "http://localhost:8000"
+faucet_pem = "faucet.pem"
+ledger_pem = "ledger.pem"
```

Comment:
As mentioned above and in the async many-client PR, I would not enforce using an identity. It can be anonymous, or we could use the identity only if this file exists.

Reply:
@fmorency ok, will work on that.